common/src/test/java/org/apache/uniffle/common/RemoteStorageInfoTest.java

```diff
@@ -119,4 +119,14 @@ public void testHashCode() {
     RemoteStorageInfo info1 = new RemoteStorageInfo(TEST_PATH, CONF_STRING);
     assertEquals(info.hashCode(), info1.hashCode());
   }
+
+  @ParameterizedTest
+  @ValueSource(strings = {"k1=v1\\,v11,k2=v2\\,v22"})
+  public void testParseReservedSymbo(String confString) {
```

Comment:
Do you refer to other systems how to solve similar problems?

Reply:
It is hard to find a splitter which is not conflict with remote storage config.
Why I use '\', because it is java escape character, I think no projects will use '\' as splitter.
I think the best way is that we should import a remote storage configuration.

Reply:
Is `Symbo` a spelling error? Or does it have some special meaning?

Reply:
Yes it is symbol, a spelling error.